apps/nameguard.io/src/components/organisms/Header.tsx

```diff
@@ -32,42 +33,43 @@ export const Header = () => {
           </div>
         </div>
         <div className="flex flex-row items-center justify-between md:gap-5 h-[40px]">
-          <div className="hidden items-center justify-center lg:flex gap-1">
+          <div className="hidden items-center justify-center lg:flex gap-0.5">
             <a
-              className="px-[10px] py-[9px] not-italic text-black font-medium text-sm leading-5 rounded-md sm:hover:bg-gray-100"
               href="https://api.nameguard.io/docs"
               target="_blank"
               rel="noopener noreferrer"
             >
-              Docs
+              <Button variant="ghost">Docs</Button>
             </a>
+
             <a
-              className="rounded-md sm:hover:bg-gray-100"
               href="https://github.com/namehash/nameguard"
               target="_blank"
               rel="noopener noreferrer"
             >
-              <div className="flex flex-row justify-between items-center gap-2 py-[7px] px-[10px]">
-                <GithubIcon className="hidden md:block text-[#0F172A] fill-current" />
-                <p className="not-italic text-black font-medium text-sm leading-5">
-                  Github
-                </p>
-              </div>
+              <IconButton
+                variant="ghost"
+                icon={
+                  <GithubIcon className="hidden md:block text-[#0F172A] fill-current" />
+                }
+              >
+                Github
+              </IconButton>
             </a>
-            <Link
-              className="px-[10px] py-[9px] not-italic text-black font-medium text-sm leading-5 rounded-md gt_mobile:hover:bg-gray-100"
-              href="/contact"
-              rel="noopener noreferrer"
-            >
-              Contact
-            </Link>
-            <CalButton className="px-4 py-2 text-sm rounded-md font-medium bg-black text-white hover:bg-gray-800 transition-all duration-300 whitespace-nowrap">
-              Discuss an integration
-            </CalButton>
+
+            <a href="/contact">
+              <Button variant="ghost">Contact</Button>
+            </a>
+
+            <CalendarButton>Discuss an integration</CalendarButton>
```

Comment:
@eduramme It looks like we should optimize something here with the horizontal spacing between buttons?

![CleanShot 2024-09-02 at 11 05 49](https://github.com/user-attachments/assets/0d82975a-18b2-418f-91ef-84d99a512ff6)

When we hover over "Contact" it looks too close to "Discuss an Integration". Same idea for other buttons in this nav bar but it is less noticeable for them because they are ghost buttons.

Alternatively, the horizontal spacing for nav buttons on namehashlabs.org looks better:

![CleanShot 2024-09-02 at 11 08 28](https://github.com/user-attachments/assets/e153723b-4d7c-4808-930c-37a88507e14e)

@eduramme Please investigate this and share your suggestion that would follow the same approach on both landing page nav bars.

Reply:
@notrab Thanks. I created a separate story for this that we could action in a separate future PR: https://app.shortcut.com/ps-web3/story/25699/create-buttongroup-ui-component

Reply:
@lightwalker-eth I reduced the spacing of the buttons there so the Header doesn't collide when we resize the screen: https://www.loom.com/share/bd579a569a3b4129acdb993916079af3?sid=865a1e66-b936-4af4-ad7f-0372bfde2226

Is it an issue if it happens as shown in the video?

Reply:
@eduramme Hey thanks for testing this and sharing the Loom video :+1:

![CleanShot 2024-09-03 at 17 37 04](https://github.com/user-attachments/assets/829f4392-b870-4e65-8575-64de364ec09a)

Suggest that we switch the top nav bar to a hamburger menu more eagerly. For example, if there isn't enough horizontal space anymore to fit the full width of the horizontal padding to the right side of the "Discuss an integration" button then we should switch to the hamburger menu variation.